conftest.py

```diff
@@ -13,8 +13,26 @@
 # limitations under the License.
 # ============================================================================
 """Blacklist for pytest."""
+
+from absl import app
+
+
 collect_ignore = [
     "discussion/",
     "setup.py",
     "tensorflow_probability/python/experimental/substrates/"
 ]
+
+def pytest_addoption(parser):
```

Comment:
I'm not too familiar with pytest, so pardon the questions: --absl-flag doesn't seem to be used anywhere in our code base. Is this for a user running a test? And we need this because by default there isn't any passing of flags from pytest to the actual test itself?

Reply:
Yes, this is for someone who wants to run pytest and pass some flags to absl. Relying on `sys.argv` doesn't seem to be an option since pytest errors on unknown flags. So I provided it a flag `--absl_flag` which can be used to pass flags down to absl.

Reply:
Gotcha, this sounds like a good change then. This doesn't immediately change anything with CI though, correct?

Reply:
I believe conftest doesn't affect CI at all. What fixed CI is removing code from test_util.

Reply:
Was it the elif or the pytest check (or both)?

Reply:
Just pytest check. Basically just found first failing commit https://github.com/tensorflow/probability/commit/ec955788a3a5fdd31e4fccee9f9106a46ebcaf0c#diff-daa5b463df9f46faa41ebafd13aa3058 and reverted it. `pytest` was in `sys.modules` because hypothesis lib imported it so it's not really good approach to test if code is run within pytest. In this case pytest guys recommend using hooks like here https://docs.pytest.org/en/latest/example/simple.html#detect-if-running-from-within-a-pytest-run So I just found a hook which is invoked when all modules are imported hence all flags are defined and can be registered.